doc/news/_preparation_next_release.md

```diff
@@ -180,6 +180,8 @@ These notes are of interest for people developing Elektra:
   - Document how to locally replicate the Docker environment used for tests.
 - <<TODO>>
 - <<TODO>>
+- Port `elektra-gcc-configure-debian-optimizations` to new build system.
```

Comment:
Maybe give a short description what the build jobs do (also in the Jenkinsfile). The name alone does not say much.

And seems like I merged in the wrong order, there is now a conflict everywhere.

Reply:
> And seems like I merged in the wrong order, there is now a conflict everywhere.

I did not pay a lot of attention when creating the entries so that they would not conflict. but the pr's are small enough that rebasing is easy.